core/src/test/scala/akka/persistence/jdbc/query/JournalDaoStreamMessagesMemoryTest.scala

```diff
@@ -54,6 +54,8 @@ abstract class JournalDaoStreamMessagesMemoryTest(configFile: String)
   behavior.of("Replaying Persistence Actor")
 
   it should "stream events" in {
+    if (newDao)
+      pending
```

Comment:
is that still true?

Reply:
Ya

Reply:
It interacts with the old schema. We could re-write it but it is tested with the old schema so not sure it is worth the effort